src/amplify-migration/amplifyUtils.ts

```diff
@@ -70,23 +90,42 @@ export async function startReleaseJob({ appId }: AmplifyAppInfo) {
 
 export async function createAmplifyApp(
   repo_name: string,
-  build_spec: string
+  build_spec: string,
+  isStagingLite: boolean = false
 ): Promise<string> {
-  const params = {
+  let redirectRules = [
+    {
+      source: "/<*>",
```

Comment:
this is a catch all regex right? why does this regex map to 404?

Reply:
this is a copy paste from [amplify's doc](https://docs.aws.amazon.com/amplify/latest/userguide/redirects.html) regarding directing users to a 404 page if not found!

Reply:
got it